.prettierignore

```diff
@@ -0,0 +1,2 @@
+# Disable Prettier for all files in this repository
+**/*
```

Comment:
this should not be here

Reply:
There's no Prettier configuration in this repo. That means for many developers, their personal Prettier settings are going to blow up your formatting.

If you are actually using Prettier, can you add a prettier config file instead so our auto-saved format matches yours?

Reply:
I am not using that, and you should not have tools like that globally enabled by default - it should default off, and only be enabled for repos where you've chosen to enable it.